frontend/src/components/landing/Contact.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+
+class Contact extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            email: '',
+            firstName: '',
+            lastName: '',
+            message: ''
+        }
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleInputChange(event) {
+        const target = event.target;
+        const value = target.value;
+        const name = target.name;
+    
+        this.setState({
+          [name]: value
+        });
+
+        console.log(event.target.value);
+      }
+
+      handleSubmit(event) {
+        console.log(this.state.value);
```

Comment:
this.state.value is undefined because this.state doesn't have a property named "value"
looks like you got confused between this.state and event.target.value

Reply:
changed this.state.value to event.target.value